data/mods/ssb/conditions.ts

```diff
@@ -211,6 +211,18 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			this.add(`c:|${getName('TheJesucristoOsAma')}|And that's how I've died for the third time, I'll go to host a game at eventos.`);
 		},
 	},
+   ut: {
+		noCopy: true,
+		onStart() {
+			this.add(`c:|${getName('ut')}|I just hope both teams have fun!`);
+		},
+		onSwitchOut() {
+			this.add(`c:|${getName('ut')}|this path is reckless`);
+		},
+		onFaint() {
+			this.add(`c:|${getName('ut')}|screaming, crying, perfect storm`);
```

Comment:
This will display as "@ut" in battle, so capitalize the "UT" in getName if you want it to display as such